vendor/shims/ua-parser-js.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-env node:true */
+/* globals define, self */
+
+(function() {
+  function vendorModule() {
+    'use strict';
+
+    return {
+      'default': self['UAParser'],
+      __esModule: true,
+    };
+  }
+
+  define('ua-parser-js', [], vendorModule);
+})();
```

Comment:
Do we want a semi-colon at the end of this?  As an aside, do we need to open an issue for our linter to lint these files as well?

Reply:
I opened https://github.com/ciena-blueplanet/ember-test-utils/issues/107

Reply:
This has been updated to comply with our linting standards.